src/pages/docs/administration/octoai/index.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2025-04-04
+modDate: 2025-04-04
+title: OctoAI
+description: How to use OctoAI
+navOrder: 100
+hideInThisSection: true
```

Comment:
`hideInThisSection` should be replaced with:

```suggestion
navSearch: false
navSitemap: false
navMenu: false
```

This is granular control over specific types of visibility.

Reply:
@mcasperson just an FYI that ⬆️ is what you'll need if you want the page to be hidden.